src/Social/Services/Comments.php

```diff
@@ -18,12 +18,12 @@ class Comments
      * @param string $message
      * @return MessageComments
      */
-    public static function add(string $messageId, string $message): MessageComments
+    public static function add(int $messageId, string $message): MessageComments
     {
-        $message = Messages::getByIdOrFail($messageId);
+        $messageData = Messages::getByIdOrFail($messageId);
```

Comment:
@arfenis why not? $message->comment($message) ?  since you always have to do add ac omment to a msg?